samples/csharp_dotnetcore/51.cafe-bot/Dialogs/Shared/StateProperties/ReservationProperty.cs

```diff
@@ -0,0 +1,322 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Recognizers.Text.DataTypes.TimexExpression;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class ReservationProperty
+    {
+        // Consts for LUIS entities.
```

Comment:
I would use the full word "constants" since they are not actually const variables